types/vision/v4/index.d.ts

```diff
@@ -0,0 +1,212 @@
+// Type definitions for vision 4.1
```

Comment:
We're on 5.x

Reply:
That's the old file, these are kept for v4 users.